gems/canvas_sanitize/lib/canvas_sanitize/canvas_sanitize.rb

```diff
@@ -90,6 +90,7 @@ def self.clean_style_value(config, value)
                    'role',
                    'lang',
                    'dir',
+                   :data,
```

Comment:
@claydiffrient I've updated the code with a comment about what `:data` does and mentioned that it should be a symbol. Let me know if that should be more clear.